src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -126,8 +127,8 @@ def get_target_path(resource_type, path, logical_name, data_directory_paths):
     for filepath in data_directory_paths:
         if filepath.type == resource_type:
             data_directory_path = filepath
-    file_type = path.split('\\')[-1].split('.')[1]
-    file_name = logical_name + '_' + str(int(time.time())) + '.' + file_type
+    file_type = '.' + path.split('\\')[-1].split('.')[1] if len(path.split('\\')[-1].split('.')) > 1 else ""
```

Comment:
Could we add some comments or samples to explain this line ?

Reply:
Sure! I can add a comment here.

Reply:
Added comment